.travis.yml

```diff
@@ -0,0 +1,23 @@
+os: 
+ - linux
+# we can't currently register URIs without a bundle and this project
+# doesn't bundle for mac.
+# - osx
+
+language: rust
+
+rust:
+ - stable
+ - nightly
+
+matrix:
+  allow_failures:
+    - rust: nightly
+
+sudo: false
+
+cache:
+  cargo: true
+
+script:
+  - cargo run --examples test
```

Comment:
Probably this (and others with this marker at the end) file is missing a `\n` at the end ? Hit enter at the EOF to make this go away (or basically do what's done for other files in this PR without this mark).